flask_monitor/metrics.py

```diff
@@ -142,12 +142,16 @@ def register_dependecy():
 def collect_dependency_time(
     app, name, rtype='http', status=200,
     is_error=False, error_message='',
-    method='GET', addr='/', **kwargs):
+    method='GET', addr='/',
+    elapsed=None,
+    start=0,
```

Comment:
@Ziul, choose only one, please. This API must be straight forward.

Reply:
Done @abilioesteves!